File: Apt/app/src/main/java/com/hencoder/apt/MainActivity.java
package com.hencoder.apt;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.hencoder.lib.Binding;
import com.hencoder.lib_annotations.BindView;

public class MainActivity extends AppCompatActivity {
  @BindView(R.id.textView) TextView textView;
  @BindView(R.id.layout) ViewGroup layout;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    Binding.bind(this);
    textView.setText("HenCoder");
    layout.setBackgroundColor(Color.CYAN);
  }
}

File: CustomViewMaterialEditText/app/src/main/java/com/hencoder/materialedittext/MainActivity.kt
package com.hencoder.materialedittext

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.core.view.postDelayed
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    met.postDelayed(3000) {
      met.useFloatingLabel = false
    }
  }
}